app/components/subheader/bootstrap.yml

```diff
@@ -1,3 +1,5 @@
 _components:
   subheader:
-    text: ''
+    text: ""
+    type: "h2"
+    link: ""
```

Comment:
Let's keep consistency by using single quotes instead of double quotes. That's the standard for our projects